src/test/java/seedu/address/logic/commands/EditDescriptorTest.java

```diff
@@ -10,7 +10,7 @@
 import seedu.address.model.book.Rating;
 import seedu.address.model.book.Status;
 import seedu.address.testutil.EditDescriptorBuilder;
-
+//@@author 592363789
```

Comment:
Remove. [You didn't code this.](https://github.com/CS2103JAN2018-F14-B4/main/pull/111/commits/0dc97cb90b9abe89eb37b1c28b22b4b1d56189a6)